docs/modules/deployment/pages/core/rpm/postgresql.adoc

```diff
@@ -1,17 +1,3 @@
-.Install language packs and set language
-[source,console]
-----
-sudo dnf install langpacks-en glibc-all-langpacks -y
-sudo localectl set-locale LANG=en_US.UTF-8
-sudo localectl
-sudo dnf makecache
-sudo dnf update -y
-sudo reboot
-sudo dnf info postgresql-server
-----
-
-NOTE: These procedures use a specific PostgreSQL version.
-Make sure you reference your current PostgreSQL version.
```

Comment:
This filehas multiple references to `postgresql-14`, contrary to the `postgresql-15` referenced in the deb instructions.